torch_frame/data/dataset.py

```diff
@@ -143,7 +145,7 @@ def __init__(
         col_stats: Dict[str, Dict[StatType, Any]],
         target_col: Optional[str] = None,
         col_to_sep: Union[str, Dict[str, str]] = ",",
-        text_embedder_cfg: Optional[TextEmbedderConfig] = None,
+        text_embedder_cfg: Optional[TextEmbedderConfigs] = None,
```

Comment:
Shall we rename all `text_embedder_cfg` to `col_to_text_embedder_cfg`?

Reply:
Or follow https://github.com/pyg-team/pytorch-frame/pull/246#discussion_r1407325685